stories/Search.stories.tsx

```diff
@@ -0,0 +1,37 @@
+import React, { FC } from 'react'
+import { ComponentStory, ComponentMeta } from '@storybook/react'
+import {
+  Input,
+  InputGroup,
+  InputLeftElement,
+  Stack,
+  Box,
+} from '@chakra-ui/react'
+import { Search2Icon } from '@chakra-ui/icons'
+
+export default {
+  title: 'Components/SearchInput',
+  component: Input,
+} as ComponentMeta<typeof Input>
+
+export const SearchInput: ComponentStory<FC> = () => (
+  <Stack>
+    <Box>
+      <h4>Simple search input</h4>
+      <InputGroup variant="search">
+        <InputLeftElement
+          pointerEvents="none"
+          children={<Search2Icon w={'0.875rem'} h={'0.875rem'} />}
```

Comment:
Minor, but is there any benefit to doing this this way instead of making `<InputLeftElement>` have literal children?

e.g.
```
<InputLeftElement pointerEvents="none">
  <Search2Icon w={'0.875rem'} h={'0.875rem'} />
</InputLeftElement>
```

Reply:
Saw this in chakra docs example, but I suppose there is no benefits, will update.